overlord/state/change.go

```diff
@@ -175,6 +175,11 @@ func (c *Change) SetStatus(s Status) {
 	c.status = s
 }
 
+// State returns the pointer to the system State
```

Comment:
why "pointer"?  I don't remember seeing that used so explicitly in the doc comment